packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/List.doc.ts

```diff
@@ -15,6 +16,15 @@ const data: ReferenceEntityTemplateSchema = {
   ],
   category: 'Components',
   related: [],
```

Comment:
For your quick reference https://github.com/Shopify/ui-extensions/pull/2008/files#r1613894669